docs/ide/editorconfig-code-style-settings-reference.md

```diff
@@ -0,0 +1,896 @@
+---
+title: ".NET Code Style Settings For Editorconfig | Microsoft Docs"
+uid: editorconfig-code-style-settings-reference.md
+ms.custom: ""
+ms.date: "12/07/2016"
+ms.reviewer: ""
+ms.suite: ""
+ms.tgt_pltfrm: ""
+ms.topic: "article"
+dev_langs:
+  - "VB"
+  - "CSharp"
+helpviewer_keywords:
+  - "editor"  
+ms.assetid:
+caps.latest.revision: 01
+author: "kaseyu"
+ms.author: "kaseyu"
+manager: "davidcsa"
+translation.priority.ht:
+  - "cs-cz"
+  - "de-de"
+  - "es-es"
+  - "fr-fr"
+  - "it-it"
+  - "ja-jp"
+  - "ko-kr"
+  - "pl-pl"
+  - "pt-br"
+  - "ru-ru"
+  - "tr-tr"
+  - "zh-cn"
+  - "zh-tw"
+---
+
+# .NET Code Style Settings For Editorconfig
+
+## Possible values
+
+`options_name = false|true : none|suggestion|warning|error`
+
+For code style option you must specify **true** (prefer this option) or **false** (do not prefer this option) a colon (`:`) and a severity (`none`, `suggestion`, `warning`, or `error`). Severity means the level of enforcement for that style you want in your code base

```

Comment:
nit: was a tiny bit confused by the use of "prefer this option" . I wonder if it'd be simpler to just say "switch this option on" and "switch this option off" (or "use this option" and "don't use this option".